cppForSwig/cryptopp/Makefile

```diff
@@ -92,9 +93,11 @@ endif
 endif
 
 ifeq ($(UNAME),Darwin)
-AR = libtool
-ARFLAGS = -static -o
-CXX = c++
+# JB: Fix this so c++ is used if compiling natively
```

Comment:
I checked and there is indeed a c++ on Linux, but it isn't necessarily clang++ on Linux. It can be made to point to clang++ on my system, but by default it points to g++. If c++ is always clang++ on OS X, then this can just be changed to CXX = clang++ for all cases where we are compiling for OS X.